[DFMD-4] Add Page Navigation to your Pages

Teecols opened the pull request:
![IMG_20210304_095911_090](https://user-images.githubusercontent.com/74029333/109941463-7c487880-7cd3-11eb-9453-4e0b5caa1685.jpg)
![Uploading IMG_20210304_095917_654.jpg…]()

Teecols commented:
![IMG_20210304_095917_654](https://user-images.githubusercontent.com/74029333/109943716-c4689a80-7cd5-11eb-89e8-744150b88d5d.jpg)

adaugochi commented on submissions/tosinKolawole/html-forms/product-page.html:
```diff
@@ -0,0 +1,49 @@
+﻿<!DOCTYPE html>
```
You don't have t create a `product-page.html` file inside the `html-forms` folder. Kindly link the `product-page.html` that is in the `html` folder

Teecols replied:
I don't really understand what I'm supposed to do. Can you kindly expatiate

adaugochi replied:
During your `html` task, you created a `product page` file and you placed that file in your `html` folder I guess. For this task, you still went ahead to create the same `product-page.html` file in your `html-forms` folder. This wrong. You don't have to create another one in your `html-forms` folder. Your product page should be located in your `html` folder not anywhere else.

adaugochi commented on submissions/tosinKolawole/html-forms/Contactus.html:
```diff
@@ -5,6 +5,22 @@
+    <div>
+        <ul>
+           <li><a href="html-forms/product-page.html">Product</a></li> 
+           <li><a href="html-forms/contact-form.html">Contact us</a></li><br>
```
You have rename your file from `contact-form.html` to `Contactus.html`

adaugochi commented:
You still haven't take correction

Teecols commented:
In your last comment, you said I changed the name of the Contact-form.html to Contactus.html. As a result, I changed it back to the previous name. Kindly specify the exact correction you're referring to so that I can effect it immediately.

Ireti203 commented:
Remove the Homepage.html file from the html-forms folder. You had earlier created a product page, place the navigation links there.

adaugochi commented:
Unattended to
